earthmover/package.py

```diff
@@ -179,19 +179,28 @@ def install(self, packages_dir):
         branch = self.error_handler.assert_get_key(self.config, 'branch', dtype=str, required=False, default=None)
 
         tmp_package_path = os.path.join(packages_dir, 'tmp_git')
-        os.mkdir(tmp_package_path)
+        os.makedirs(tmp_package_path, exist_ok=True)
+
+        try:
+            if branch:
```

Comment:
Another petty code-cleanliness update. I'd prefer this conditional block to be rewritten as follows:

```
if branch:
    command = ["git", "clone", "-b", branch, source_path, "."]
else:  # If branch is not specified, default working branch is used
    command = ["git", "clone", source_path, "."]

subprocess.run(command, cwd=tmp_package_path, timeout=git_auth_timeout)
```